### ISSUE TITLE
Build error at rust version

### ISSUE THREAD

**Issue by satoren** [versatica/mediasoup#754]
## Bug Report

An error occurs in the rust version since January 11, 2022. This does not occur in the node version.
It seems to be caused by an update of the meson package in pip.
https://pypi.org/project/meson/#history
So the following workaround will allow the build to pass.

```diff
diff --git a/worker/Makefile b/worker/Makefile
index c67982c2..2be6a93d 100644
--- a/worker/Makefile
+++ b/worker/Makefile
@@ -63,7 +63,7 @@ ifeq ($(wildcard $(PIP_DIR)),)
                echo "Installation failed, likely because PIP is unavailable, if you are on Debian/Ubuntu or derivative please install the python3-pip package"
        # Install `meson` and `ninja` using `pip` into custom location, so we don't
        # depend on system-wide installation.
-       $(PYTHON) -m pip install --upgrade --target=$(PIP_DIR) $(PIP_BUILD_BINARIES) meson ninja
+       $(PYTHON) -m pip install --upgrade --target=$(PIP_DIR) $(PIP_BUILD_BINARIES) meson==0.60.3 ninja
 endif
 
 setup: meson-ninja
```

### Your environment

- Operating system: Debian bullseye (Linux aarch64 GNU/Linux) It also happens with CI on Github Actions.
- Node version: N/A
- npm version: N/A
- gcc/clang version: gcc version 10.2.1 20210110 (Debian 10.2.1-6)
- mediasoup version: `tag: 3.9.4`
- mediasoup-client version: N/A

### Issue description

```
 cargo test
```

<details>

```bash

   Compiling mediasoup v0.9.2 (/workspaces/mediasoup/rust)
     Running `CARGO=/usr/local/rustup/toolchains/1.56.1-aarch64-unknown-linux-gnu/bin/cargo CARGO_CRATE_NAME=mediasoup CARGO_MANIFEST_DIR=/workspaces/mediasoup/rust CARGO_PKG_AUTHORS='Nazar Mokrynskyi <nazar@mokrynskyi.com>' CARGO_PKG_DESCRIPTION='Cutting Edge WebRTC Video Conferencing in Rust' CARGO_PKG_HOMEPAGE='' CARGO_PKG_LICENSE=ISC CARGO_PKG_LICENSE_FILE='' CARGO_PKG_NAME=mediasoup CARGO_PKG_REPOSITORY='https://github.com/versatica/mediasoup' CARGO_PKG_VERSION=0.9.2 CARGO_PKG_VERSION_MAJOR=0 CARGO_PKG_VERSION_MINOR=9 CARGO_PKG_VERSION_PATCH=2 CARGO_PKG_VERSION_PRE='' CARGO_PRIMARY_PACKAGE=1 LD_LIBRARY_PATH='/workspaces/mediasoup/target/debug/deps:/usr/local/rustup/toolchains/1.56.1-aarch64-unknown-linux-gnu/lib:/usr/local/rustup/toolchains/1.56.1-aarch64-unknown-linux-gnu/lib' rustc --crate-name mediasoup --edition=2018 rust/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi --crate-type lib --emit=dep-info,metadata,link -C embed-bitcode=no -C debuginfo=2 -C metadata=0d0f98d2af3c0bb5 -C extra-filename=-0d0f98d2af3c0bb5 --out-dir /workspaces/mediasoup/target/debug/deps -C incremental=/workspaces/mediasoup/target/debug/incremental -L dependency=/workspaces/mediasoup/target/debug/deps --extern async_channel=/workspaces/mediasoup/target/debug/deps/libasync_channel-0ba8c2701b254e75.rmeta --extern async_executor=/workspaces/mediasoup/target/debug/deps/libasync_executor-022ecc409640cf7f.rmeta --extern async_lock=/workspaces/mediasoup/target/debug/deps/libasync_lock-b430be62a1c87fb1.rmeta --extern async_oneshot=/workspaces/mediasoup/target/debug/deps/libasync_oneshot-baa8de65da7f98d2.rmeta --extern async_trait=/workspaces/mediasoup/target/debug/deps/libasync_trait-ac99dabd1b4714c3.so --extern atomic_take=/workspaces/mediasoup/target/debug/deps/libatomic_take-00eefcd7fd319de9.rmeta --extern event_listener_primitives=/workspaces/mediasoup/target/debug/deps/libevent_listener_primitives-f84cc9f3f13d6e6d.rmeta --extern fastrand=/workspaces/mediasoup/target/debug/deps/libfastrand-d3c20ccc4a422bdc.rmeta --extern futures_lite=/workspaces/mediasoup/target/debug/deps/libfutures_lite-1b6bb7083d3b2012.rmeta --extern h264_profile_level_id=/workspaces/mediasoup/target/debug/deps/libh264_profile_level_id-497b50c23d4089ca.rmeta --extern hash_hasher=/workspaces/mediasoup/target/debug/deps/libhash_hasher-5d1f64c4441e3dd0.rmeta --extern log=/workspaces/mediasoup/target/debug/deps/liblog-e1d8303c4b2059c5.rmeta --extern lru=/workspaces/mediasoup/target/debug/deps/liblru-0cce17966b48da1a.rmeta --extern mediasoup_sys=/workspaces/mediasoup/target/debug/deps/libmediasoup_sys-da259e3a089b30e2.rmeta --extern nohash_hasher=/workspaces/mediasoup/target/debug/deps/libnohash_hasher-603d39cb0af2594d.rmeta --extern once_cell=/workspaces/mediasoup/target/debug/deps/libonce_cell-f2965451202c2afd.rmeta --extern parking_lot=/workspaces/mediasoup/target/debug/deps/libparking_lot-3a649dee013fea0b.rmeta --extern regex=/workspaces/mediasoup/target/debug/deps/libregex-eb70674307b71299.rmeta --extern serde=/workspaces/mediasoup/target/debug/deps/libserde-8c0104ff40c3e8a2.rmeta --extern serde_json=/workspaces/mediasoup/target/debug/deps/libserde_json-b4a30fb57d9ac7bf.rmeta --extern serde_repr=/workspaces/mediasoup/target/debug/deps/libserde_repr-5d7195edce1f86a3.so --extern thiserror=/workspaces/mediasoup/target/debug/deps/libthiserror-d082eecd957f7d67.rmeta --extern uuid=/workspaces/mediasoup/target/debug/deps/libuuid-6f8588a8f35b6fed.rmeta -L native=/usr/lib/gcc/aarch64-linux-gnu/10/ -L native=/workspaces/mediasoup/target/debug/build/mediasoup-sys-f12ca51c5adcb64c/out`
error: failed to build archive: 'libmediasoup-worker.a.p/src_lib.cpp.o': No such file or directory

error: could not compile `mediasoup-sys` due to previous error

Caused by:
  process didn't exit successfully: `CARGO=/usr/local/rustup/toolchains/1.56.1-aarch64-unknown-linux-gnu/bin/cargo CARGO_CRATE_NAME=mediasoup_sys CARGO_MANIFEST_DIR=/workspaces/mediasoup/worker CARGO_PKG_AUTHORS='Nazar Mokrynskyi <nazar@mokrynskyi.com>' CARGO_PKG_DESCRIPTION='FFI bindings to C++ libmediasoup-worker' CARGO_PKG_HOMEPAGE='' CARGO_PKG_LICENSE=ISC CARGO_PKG_LICENSE_FILE='' CARGO_PKG_NAME=mediasoup-sys CARGO_PKG_REPOSITORY='https://github.com/versatica/mediasoup/tree/v3/worker' CARGO_PKG_VERSION=0.3.1 CARGO_PKG_VERSION_MAJOR=0 CARGO_PKG_VERSION_MINOR=3 CARGO_PKG_VERSION_PATCH=1 CARGO_PKG_VERSION_PRE='' CARGO_PRIMARY_PACKAGE=1 LD_LIBRARY_PATH='/workspaces/mediasoup/target/debug/deps:/usr/local/rustup/toolchains/1.56.1-aarch64-unknown-linux-gnu/lib:/usr/local/rustup/toolchains/1.56.1-aarch64-unknown-linux-gnu/lib' OUT_DIR=/workspaces/mediasoup/target/debug/build/mediasoup-sys-f12ca51c5adcb64c/out rustc --crate-name mediasoup_sys --edition=2018 worker/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi,artifacts --crate-type lib --emit=dep-info,metadata,link -C embed-bitcode=no -C debuginfo=2 -C metadata=da259e3a089b30e2 -C extra-filename=-da259e3a089b30e2 --out-dir /workspaces/mediasoup/target/debug/deps -C incremental=/workspaces/mediasoup/target/debug/incremental -L dependency=/workspaces/mediasoup/target/debug/deps -L native=/usr/lib/gcc/aarch64-linux-gnu/10/ -L native=/workspaces/mediasoup/target/debug/build/mediasoup-sys-f12ca51c5adcb64c/out -l static=stdc++ -l static=mediasoup-worker` (exit status: 1)
warning: build failed, waiting for other jobs to finish...
error: build failed
```

</details>

**Comment by nazar-pc** [comment]
I can reproduce it, weird

**Comment by nazar-pc** [comment]
Fix is in #755

**Comment by nazar-pc** [comment]
New patch version of the worker crate is out, should fix this issue
